jsonschema/compat.py

```diff
@@ -40,6 +43,9 @@ def urlsplit(url):
     return SplitResult(scheme, netloc, path, query, fragment)
 
 
+DefragResult = namedtuple('DefragResult', 'url fragment')
```

Comment:
Is this to just gain attribute access? If so I'd prefer an actual class, namedtuple is a plague.